### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by danni**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,77 @@
+APP = ask_izzy
+REPO = contyard.office.infoxchange.net.au/$(APP)
```
Comment (DanielHeath) [inline Makefile:2]: Is that where the source is hosted internally?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,77 @@
+APP = ask_izzy
```
Comment (DanielHeath) [inline Makefile:1]: Should we use the Makefile in place of a directory full of scripts?

File: Makefile (hunk 1 above)
Comment (danni) [inline Makefile:2]: That's the internal Docker container yard that Straddle Carrier uses to load containers.

File: Makefile (hunk 2 above)
Comment (danni) [inline Makefile:1]: Can do? Flexible.

**Pull request merged**